src/main/java/com/iexec/core/replicate/Replicate.java

```diff
@@ -39,6 +40,27 @@ public ReplicateStatus getCurrentStatus() {
         return this.getLatestStatusChange().getStatus();
     }
 
+    @JsonIgnore
+    public ReplicateStatus getLastRelevantStatus() {
```

Comment:
I think this method is very dangerous :/

Reply:
But we need it to find out in what status the replicate was before trying to RECOVER (one or more times) and before being lost.
For instance: this is a typical exp: CONTRIBUTING > WORKER_LOST > RECOVERING > RECOVERING